nav2_minimal_tb4_sim/launch/spawn_tb4.launch.py

```diff
@@ -106,24 +106,18 @@ def generate_launch_description():
         arguments=['/rgbd_camera/depth_image'])
 
     spawn_model = Node(
-        condition=IfCondition(use_simulator),
```

Comment:
This is something I personally needed since I'm running with `use_simulator=false` so that another part of my launch file can launch the simulator. I personally feel this parameter doesn't make too much sense here, I assume that if you run a launch file called `spawn_tb4` you want to spawn a turtlebot? In the previous behavior it would run a bunch of other simulator nodes (i.e. the bridges) but not actually spawn a turtlebot which is a bit unexpected

Reply:
This is so that someone else can spawn a simulation with the robot in it and this launch file would then provide the bridges for the topics over to ROS 2 which this launch file also provides. So whether a robot is spawned in a SDF or here, we can get the bridge going